Components/Image/image.css

```diff
@@ -0,0 +1,10 @@
+.img-container {
+  width: 80vw;
+  height: 10vh;
```

Comment:
Giving definite value to container will cause trouble when someone or you later try to use it without specifying other values

Reply:
Actually , I never used this class. Thanks for pointing out. Will remove it